src/slider.ts

```diff
@@ -0,0 +1,33 @@
+import { HTML } from 'bigtest';
+
+export default HTML.extend('slider')
+  .selector('span[role="slider"]')
+  .filters({
+    ariaValueNow: (el) => {
+      return el.getAttribute('aria-valuenow')
+    },
+    dataIndex: (el) => el.getAttribute('data-index'),
+    index: (el) => {
```

Comment:
Between Line 9 and Line 10. These are two ways to get the thumb or thumbs on slider when there are multiple thumbs on the slider they are given an index. The component library adds the data-index.